src/Database/Auth/ApiKeyAuthenticator.ts

```diff
@@ -69,7 +70,7 @@ export class ApiKeyAuthenticator {
 
     return tryAuthenticate()
       .then((response: IResponse) => {
-        const body = response.body;
+        const body = response.body as IHash;
```

Comment:
object isn't equivalent to 'object with any properties': http://take.ms/TNvxz
for declare  'object with any properties' i should explicitly declare it like {[property: string]: any}
as i meet with this type in many places i created IHash interface for don't declare {[property: string]: any} everywhere